imports/ui/JapaneseNewGame.js

```diff
@@ -801,14 +802,13 @@ function push_dealin_hand(template) {
 		Session.set("north_riichi_sum", Number(Session.get("north_riichi_sum")) + 1);
 	}
 
-	eastDelta  += dealin_delta(points, fu, "east", winnerWind, loserWind)  +
-				  rewardRiichiSticks(riichiSum, "east", winnerWind);
-	southDelta += dealin_delta(points, fu, "south", winnerWind, loserWind) +
-				  rewardRiichiSticks(riichiSum, "south", winnerWind);
-	westDelta  += dealin_delta(points, fu, "west", winnerWind, loserWind)  +
-				  rewardRiichiSticks(riichiSum, "west", winnerWind);
-	northDelta += dealin_delta(points, fu, "north", winnerWind, loserWind) +
-				  rewardRiichiSticks(riichiSum, "north", winnerWind);
+	allDelta = PointCalculationUtils.jpn_dealin_delta(points, fu, winnerWind, loserWind, riichiSum);
+	Session.set("free_riichi_sticks", 0);
```

Comment:
1. This is redundant as it exists in PointCalculationUtils (I would remove it from there personally)
2. This is located in a different location than in self_draw.  Should be consistent.